docs/tm/registry.md

```diff
@@ -1,8 +1,8 @@
-## Docker registry
+## Docker Registry
 
 Docker images are used to run functions code in Knative services. This means that image registry is important part of service deployment scheme. Depending on type of service, Knative controller may either only pull or also push service image from and to registry. Triggermesh CLI provides simple configuration interface to setup registry address and user access credentials.
 
-### Service from pre-build image
+### Service from Pre-Build Image
```

Comment:
Not directly related to the capitalization issue, but this should be `Pre-Built` instead of `Pre-Build`

Reply:
Thank you :) I changed this